Apps/Database/src/Delegates/DBUserDelegateDelegate.cs

```diff
@@ -92,5 +92,19 @@ public DBResult<UserDelegate> Delete(string ownerId, string delegateId, bool com
         {
             throw new System.NotImplementedException();
         }
+
+        /// <inheritdoc />
+        public bool Find(string ownerId, string delegateId)
```

Comment:
i think "exists" make more sense as a name!

Reply:
Agreed, I said Find but we're not returning the object so exists makes more sense.

Reply:
Updated the name to Exists